bin/onchange-nfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
+then
+  echo "nfs is healthy, mounting uploads directory...."
+  mount -t nfs -v -o nolock,vers=3 nfs:/exports /var/www/html/content/uploads
+else
```

Comment:
it looks like setting upload_max_filesize to 0 will make it unlimited, but there is the "file_uploads" option that setting to 0 will completely disable uploads over http, seems like that might work well. I will do some testing with that.

I had some other ideas to prevent files from being written to the uploads directory while it is not mounted (just chmod it to prevent writes to that dir), but they beg another question. How do we feel about the fact that the WP container is currently running as root? I had attempted to change this a one point, but I think it hindered some of my troubleshooting efforts. Everything is in a good place now, so we could look at changing it again. But I guess since we are mounting an NFS export now it will need to be running as root...